app/src/main/java/com/theopensourcefamily/chessclock/BoardState.kt

```diff
@@ -0,0 +1,6 @@
+package com.theopensourcefamily.chessclock
+
+sealed class BoardState
+object Stopped : BoardState()
+object WhiteRunning : BoardState()
+object BlackRunning : BoardState()
```

Comment:
Hmmm i have doubt about squashing blacks + whites running states, it could be a boolean inside the sealed class, but that would mean treat that comparison inside the view... maybe i like it more the way you did it, what do you think?

Reply:
Ok i've thought about it a little more, and I see your approach definitely better than a single state for both blacks and whites 🚀